katello-selinux-relabel

```diff
@@ -2,25 +2,6 @@
 
 # relabel foreman
 /sbin/restorecon -ri $* /usr/share/foreman \
-  /usr/share/katello \
-  /var/lib/foreman \
```

Comment:
I'd argue these should stay or all of the foreman specifics should move to a `foreman-selinux-relabel`. For now I'd just remove all of the 1.8 bits and ruby193 since both are no longer used.

Reply:
Currently in Foreman:

```
   /usr/share/foreman
  /var/lib/foreman \
  /var/run/foreman \
  /run/foreman \
  /var/log/foreman \
  /etc/foreman \
  /etc/puppet/node.rb \
  /etc/puppetlabs/puppet/node.rb \
  /etc/sysconfig/foreman* \
  /etc/rc.d/init.d/foreman* \
  /etc/logrotate.d/foreman* \
  /etc/cron.d/foreman* \
  /usr/lib{64,}/ruby/gems/1.8/gems/passenger-* \
  /usr/lib{64,}/gems/ruby/passenger-*/agents \
  /usr/lib{64,}/ruby/site_ruby/1.8/x86_64-linux/agents \
  /usr/share/passenger/helper-scripts \
  /usr/lib{64,}exec/passenger \
  /var/run/rubygem-passenger
```

Everything can go, so +1 from me.